src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx).
+
+You can update your project's version of Astro to the latest version using your package manager. If you're using Astro integrations, please also update those to the latest version.
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # updates the astro dependency:
+  npm upgrade astro
+  # or, to update all dependencies:
+  npm upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # updates the astro dependency:
+  pnpm upgrade astro
+  # or, to update all dependencies:
+  pnpm upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # updates the astro dependency:
+  yarn upgrade astro
+  # or, to update all dependencies:
+  yarn upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+This guide includes major highlights and instructions on how to handle breaking changes when upgrading from v1.x and v2.0-beta releases. See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+
+## Astro 2.0 Updates
+
+### Updated: Node Version >=16.12.0
+
+The minimum supported version of NodeJS is now 16.12.0.
+
+
+### Updated: Vite 4
+
+Astro v2.0 has upgraded from Vite 3 to [Vite 4](https://vitejs.dev/). We've handled most of the upgrade for you inside of Astro; however, some subtle Vite behaviors may still change between versions. 
```

Comment:
For both this paragraph and the Node one above it, I'd add 1 sentence of context. Something like:

> Node 14 is scheduled to reach its End of Life in April 2023. Astro 2.0 drops Node 14 support entirely, so that all Astro users can take advantage of Node's more modern features. Astro 2.0 requires Node.js 16.12.0 or later.

> The latest version of Vite (Vite 4.0) was released in December 2022. Astro v2.0 includes an upgrade to this latest version of Vite. We've handled most of the breaking changes for you inside of Astro; however, some subtle Vite behaviors can still change between versions.